src/main/java/pro/sky/HomeWork28/service/DepartmentServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package pro.sky.HomeWork28.service;
+
+import org.springframework.stereotype.Service;
+import pro.sky.HomeWork28.model.Employee;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class DepartmentServiceImpl implements DepartmentService {
+
+    private final EmployeeServiceImpl employeeService;
+
+    public DepartmentServiceImpl(EmployeeServiceImpl employeeService) {
```

Comment:
```suggestion
    public DepartmentServiceImpl(EmployeeService employeeService) {
```
надо использовать интерфейсные ссылки